lib/iotaUtils.js

```diff
@@ -135,6 +135,9 @@ function findOrCreate(deviceId, transport, group, callback) {
             if (!newDevice.transport) {
```

Comment:
CNR entry would be needed?

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/340/commits/e1556671e5b766dcd6a4309ecc11277088b2554e